### ISSUE TITLE
Shutdown option is stuck

### ISSUE THREAD

**Issue by xZakko** [Bionus/imgbrd-grabber#1783]
After having used the automatic "shutdown" option after finishing the download of images, every time the program starts up I have to make sure to change in back to "nothing" again. It keeps going back to "shutdown" after the program has been closed which results in constantly having my PC shutdown because I forgot to change it to "nothing" again.

**Comment by Bionus** [comment]
So you mean you want it *not* to remember your choice the next time, but only for the "Shutdown" option?

**Comment by xZakko** [comment]
No, I mean the batch download has "when the download is finished" options: "Do nothing", "Close window", "Open CD tray", "Open destination folder", "Play a sound" and "Shutdown."

It used to default to "Do nothing", but after having made use of the "Shutdown" option once, it always defaults to "Shutdown" no matter what I do or change it to, resulting in accidental shutdowns of my PC.

**Comment by xZakko** [comment]
Oh I understand what you mean now. It's fine that it remembers the last choice, however it's currently *stuck* on "shutdown" which is my issue. My last used option is "Do nothing" but it changes back to "Shutdown" after the program has been closed even after properly finishing a batch download on the "Do nothing" option.
